catalog/dags/common/licenses/licenses.py

```diff
@@ -189,27 +217,33 @@ def _get_valid_cc_url(license_url) -> str | None:
 
 
 def get_license_info_from_license_pair(
-    license_, license_version, pair_map=REVERSE_LICENSE_PATH_MAP
-) -> tuple[str | None, str | None, str | None]:
+    license_: str | None, license_version: str | int | float | None, pair_map=None
+) -> tuple[str, str, str] | None:
     """
     Validate a given license pair, and derive a license URL from it.
 
     Returns both the validated pair and the derived license URL.
     """
-    string_version = _ensure_license_version_string(license_version)
+    if pair_map is None:
+        pair_map = REVERSE_LICENSE_PATH_MAP
+    string_version = _ensure_license_version_string_or_none(license_version)
```

Comment:
`string_version` could be None, but we were not checking for it before. Even licenses/publicdomain has a non-None version ("N/A")